.github/workflows/e2e-backend-switching-reuseable.yml

```diff
@@ -50,7 +50,7 @@ jobs:
           MACHINE_TYPE: "c4.xlarge"
           SCALER_MIN: "1"
           SCALER_MAX: "2"
-          RETRY_ATTEMPTS: "2"
+          RETRY_ATTEMPTS: "1"
```

Comment:
Why set this down?

Reply:
As we have quota limits, so retry may cause issues. Also, it will fail sooner with single retry.